examples/rcat/c/rcat.c

```diff
@@ -131,15 +127,14 @@ int main(int argc, char *argv[]) {
 
         rasta_connection_config connection = {
             .config = &config,
-            .rasta_id = ID_R,
             .transport_sockets = toServer,
             .transport_sockets_count = sizeof(toServer) / sizeof(toServer[0])};
 
         rc = rasta_lib_init_configuration(&config, &connection, 1, LOG_LEVEL_DEBUG, LOGGER_TYPE_CONSOLE);
 
         rasta_bind(rc);
 
-        struct rasta_connection *c = rasta_connect(rc, ID_R);
+        struct rasta_connection *c = rasta_connect(rc, config.general.rasta_id_remote);
```

Comment:
We should refactor `rasta_connect`/`sr_connect` to also read the remote ID from the config (at least when we finally stop supporting multiple parallel connections, see #74)